index.html

```diff
@@ -64,7 +64,7 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
 
```

Comment:
### Your Button

![image](https://user-images.githubusercontent.com/93039359/195178231-c069029c-8332-40d0-98f1-ec83b335a92a.png)

Your button doesn't have a 4.5:1 color contrast ratio. Kindly make it to follow the requrements and to make your portfolio accessible to color blind users :blush:

Reply:
Thanks. fixed

Reply:
Hover was updated already.  We added a darker purple.